example-app/SantokuApp/src/fixtures/msw/utils/format2Iso8601.ts

```diff
@@ -1,3 +1,8 @@
 export const format2Iso8601 = (d: Date): string => {
-  return `${d.getFullYear()}-${d.getMonth() + 1}-${d.getDate()}T${d.getHours()}:${d.getMinutes()}:${d.getSeconds()}Z`;
+  const month = String(d.getMonth() + 1).padStart(2, '0');
+  const date = String(d.getDate()).padStart(2, '0');
+  const hours = String(d.getHours()).padStart(2, '0');
+  const minutes = String(d.getMinutes()).padStart(2, '0');
+  const seconds = String(d.getSeconds()).padStart(2, '0');
+  return `${d.getFullYear()}-${month}-${date}T${hours}:${minutes}:${seconds}Z`;
```

Comment:
Why not use [`Date#toISOString`](https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString)?